scripts/create-missing-test-files.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2043
+for folder in tests; do
```

Comment:
Why is the cycle needed?

Reply:
It's a carryover from the cnf-cert-test suite script that has multiple folders with Go code in them.  I'll remove it because all I care about is `/tests`.